src/redux/books/books.js

```diff
@@ -0,0 +1,29 @@
+// books.js
+
+// Actions
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```

Comment:
The value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so Kindly add them following the ducks pattern. :+1: it will be something like: :arrow_down:

```
const ADD_BOOK = bookstore/books/ADD_BOOK 
const REMOVE_BOOK = bookstore/books/REMOVE_BOOK
```